README.md

```diff
@@ -58,6 +58,13 @@ Create a client using access tokens:
 var client = new Intercom.Client({ token: 'my_token' });
 ```
 
+If you are using the EU service you can configure like the following:
+
+```node
+var client = new Intercom.Client({token: 'my_token' });
```

Comment:
Can you add a space between the opening bracket and `token:`

Reply:
done.